src/client/components/pages/entities/title.js

```diff
@@ -35,7 +35,6 @@ function EntityTitle({entity}) {
 		<div>
 			<h1>{label}{disambiguation}</h1>
 			{aliases}
-			<hr/>
```

Comment:
To allow displaying author credits immediately below edition title without intervening rule.